CHANGES.md

```diff
@@ -8,6 +8,9 @@ Change Log
 
 ### 1.18 - 2016-02-01
 
+* Breaking changes
+    * Removed `OpenStreetMapImageryProvider`
```

Comment:
In following the others, just add another sentence: "Use `createOpenStreetMapImageryProvider` instead."
Also remove the white space.